.gitignore

```diff
@@ -0,0 +1,4 @@
+/_build
```

Comment:
I'm not too familiar with how escript works, but I think that you're right: it should just need the escript file, and Erlang installed in the target environment.

I was just curious about the `beam` file in the top directory, as it seems out of place, when most of the compiled binaries would be in `_build`

Makes sense to have something runnable available to users without them having to build it themselves - especially escript - which is pretty portable so long as Erlang is available.

As for coverage - it's definitely a positive that it's there, but normally it wouldn't be checked it, as it can be generated easily again when needed, and makes the repository larger than it needs to be.